content/docs/for-developers/sending-email/getting-started-smtp.md

```diff
@@ -13,71 +13,150 @@ navigation:
-1. Optionally, add a mail-to header to add the name and email address of the recipient to the email header:
-    <br>`To: <<NAME>> <<EMAIL>>`
-    <br>Press `[Enter]`
-1. Next, add a from header to add the name and email address of the sender to the email header - if not included, SendGrid blocks your email because it doesn’t follow RFC 5322 compliance guidelines:
-    <br>`From: <<NAME>> <<EMAIL>`
-    <br>Press `[Enter]`
-1. Include a subject line:
-    <br>`Subject: <<EMAIL_SUBJECT>>`
-    <br>Press `[Enter]`
-1. Add the content of the message:
-    <br>`"<<MESSAGE>>"`. For example: `“This is a test for the SMTP relay."`
-    <br>Press `[Enter]`
-1. Finally, send the email by typing a period, and then pressing enter:
-    <br>`.`
-    <br>Press `[Enter]`
-    <br>The mail server returns `250 Ok: queued as A1AywHK7T_itbGWaASw2YQ` - This means the email has been queued to send. This queue moves very quickly.
-1. Exit the Telnet connection with: `quit`.
+1. Start a Telnet session by typing the following in the terminal:
+
+```shell
+TELNET smtp.sendgrid.net 25
+```
+
+SendGrid accepts unencrypted and TLS connections on ports **25**, **587**, & **2525**. You can also connect via SSL on port **465**. Many hosting providers and ISPs block port 25 as a default practice. If your Telent session continually times out or will not connect using port 25, it is likely that your ISP or hosting provider is blocking the port. You can contact your host/ISP to find out which ports are open for outgoing SMTP relay. We recommend using port 587 to avoid any rate limiting that your server host may apply.
+
+2. Once you successfully connect to SendGrid, log in to the server by typing the following:
+
+```shell
+AUTH LOGIN
+```
+
+The mail server should respond with `334 VXNlcm5hbWU6`, which is a Base64 encoded request for your username.
+
+3. Input `YXBpa2V5` and press **Enter** on your keyboard. Twilio SendGrid requires you to authenticate using an API key. When using Basic Authentication and an API key, you must use the string `apikey` in place of your account username. The string `apikey` is `YXBpa2V5` when Base64 encoded, which is why we use it in this step.
+
+The mail server should respond with `334 UGFzc3dvcmQ6`. This response is a Base64 encoded request for your password (your API Key).
+
+4. Enter your Base64 converted API key in the next line as the password and press **Enter**.
+
+The mail server should respond with `235 Authentication successful`. Getting this far indicates that your connection to `smtp.sendgrid.net` over the chosen port is open and that your API key is valid.
+
+5. Next, add the email that you’re sending from using the SMTP MAIL FROM command and press **Enter**.
+
+```shell
+MAIL FROM:<<SENDER_EMAIL>>
+```
+
+The mail server should respond with `250 Sender address accepted`.
+
+6. Add the email that you’re sending to using the SMTP RCPT TO command and press **Enter**.

```

Comment:
Do we want to mention here also that you can add more than one RCTP TO?

Reply:
Yeah, good call-out. I'll add that.